### DISCUSSION TITLE
Weird installation bug if you give a node a bad hostname

### DISCUSSION THREAD

**Discussion by kernelmustard** [Security-Onion-Solutions/securityonion#5743] [General] [answered]
# Install
Install is from an offical 2.3.70-WAZUH ISO onto an airgapped, distributed sensor. I will test it on the new release when I get a chance.
# Issue
During so-setup, I set the hostname to 'securityonion-sensor'. When I finish configuring and it begins the second install, it will run for awhile and then error out and give a ValueError and say that I'm missing a quote.
The workaround is easy, just name it something else. But I thought this was interesting enough to share.

**Comment by weslambert** [comment, marked as answer]
I've named my boxes things like `securityonion-testing` , etc before without issue.  Please try naming it `securityonion-testing` during setup and see if you still experience the same issue (wonder if it is at all related, or if it could be related to `sensor`).  Also, we have the .80 release available now if you would like to see if that makes any difference for you.
